### PR TITLE
remove concat for single iteration TI

### PR DESCRIPTION
**Pull request by esmirno**
concat layer looks not needed when we unroll TI with single iteration only - please review.

### CODE REVIEW

File: inference-engine/src/legacy_api/src/net_pass.cpp [hunk 1]
```diff
@@ -527,6 +527,12 @@ bool unrollTI(CNNLayerPtr cur, ICNNNetwork& net) {
     for (int i = 0; i < first_class.size(); i++) {
         auto& rule = first_class[i];
         auto out_data = ti->outData[rule.from];
+        
+        if (num == 1) {
```
Comment (itikhono) [inline inference-engine/src/legacy_api/src/net_pass.cpp:531]: Should we also remove Split layer for `num == 1` case?

**Comment by GlebKazantaev** [comment]
@itikhono are you going to handle this cases in ngraph based unroll TI transformation?

**Comment by itikhono** [comment]
> @itikhono are you going to handle this cases in ngraph based unroll TI transformation?

Yes, I am.

File: inference-engine/src/legacy_api/src/net_pass.cpp (hunk 1 above)
Comment (esmirno) [reply inference-engine/src/legacy_api/src/net_pass.cpp:531]: i would recommend that as well.

**Comment by itikhono** [comment]
The Unroll TI transformation (and other TI related transformations) was implemented on ngraph side, this PR looks out of date.
@GlebKazantaev @esmirno Can we close it?

**Comment by ilya-lavrenov** [comment]
Legacy code => close

**Pull request closed without merging**